components/style/properties/longhand/box.mako.rs

```diff
@@ -817,8 +817,8 @@ ${helpers.single_keyword("-moz-appearance",
                             scrollbar-horizontal scrollbar-small scrollbar-vertical scrollbarbutton-down
                             scrollbarbutton-left scrollbarbutton-right scrollbarbutton-up scrollbarthumb-horizontal
                             scrollbarthumb-vertical scrollbartrack-horizontal scrollbartrack-vertical searchfield
-                            separator spinner spinner-downbutton spinner-textfield spinner-upbutton splitter statusbar
-                            statusbarpanel tab tabpanel tabpanels tab-scroll-arrow-back tab-scroll-arrow-forward
+                            separator inner-spin-button spinner spinner-downbutton spinner-textfield spinner-upbutton splitter
```

Comment:
These were ordered, any chance to keep them that way? Also, this probably needs a bindings update.

Reply:
Yep, my mistake. I have fixed it.